client/devdocs/design/app-components.jsx

```diff
@@ -1,25 +1,29 @@
 /**
 * External dependencies
 */
-var React = require( 'react' ),
-	page = require( 'page' ),
-	toTitleCase = require( 'to-title-case' ),
-	trim = require( 'lodash/trim' );
+import React from 'react';
+import page from 'page';
+import toTitleCase from 'to-title-case';
+import trim from 'lodash/trim';
 
 /**
  * Internal dependencies
  */
-var SearchCard = require( 'components/search-card' ),
-	CommentButtons = require( 'components/comment-button/docs/example' ),
-	PostSelector = require( 'my-sites/post-selector/docs/example' ),
-	LikeButtons = require( 'components/like-button/docs/example' ),
-	FollowButtons = require( 'components/follow-button/docs/example' ),
-	Sites = require( 'lib/sites-list/docs/example' ),
-	SitesDropdown = require( 'components/sites-dropdown/docs/example' ),
-	Theme = require( 'components/theme/docs/example' ),
-	PostSchedule = require( 'components/post-schedule/docs/example' ),
-	HeaderCake = require( 'components/header-cake' ),
-	Collection,
+import HeaderCake from 'components/header-cake';
+import SearchCard from 'components/search-card';
+
+// Example App Components
```

Comment:
The new import order can be kept, but, for consistency, this comment and its accompanying newline should be removed.